packages/booking/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "@booking-services/booking",
+  "version": "1.0.0",
+  "description": "This package contains all bussiness logic from booking squad",
+  "author": "Booking Squad",
+  "homepage": "https://github.com/Platzi-Master-C9/backend-services/tree/main/packages/booking#readme",
+  "license": "ISC",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
```

Can you change this to `src`?
```suggestion
    "src"
```